client/console/src/main/java/org/apache/syncope/client/console/panels/ConnObjectListViewPanel.java

```diff
@@ -139,12 +155,26 @@ public void onClick(final AjaxRequestTarget target) {
 
             @Override
             protected Component getValueComponent(final String key, final ConnObjectTO bean) {
-                Optional<AttrTO> attrTO =
-                        bean.getAttrs().stream().filter(object -> object.getSchema().equals(key)).findAny();
-
-                return !attrTO.isPresent() || attrTO.get().getValues().isEmpty()
-                        ? new Label("field", StringUtils.EMPTY)
-                        : new CollectionPanel("field", attrTO.get().getValues());
+                if (StringUtils.equals(key, "Status")) {
+                    ReconStatus status = recRestClient.status(new ReconQuery.Builder(anyType, resource.getKey()).
+                            connObjectKeyValue(bean.getAttr(ConnIdSpecialName.UID).get().getValues().get(0)).build());
+
+                    return status.getOnSyncope() == null
+                            ? StatusUtils.getLabel("field", "notfound icon", "Not found", Constants.NOT_FOUND_ICON)
+                            : StatusUtils.getLabel("field", "present_icon",
+                                    status.getMatchType() == MatchType.LINKED_ACCOUNT
```

Comment:
Please avoid reading back again each match only for extracting the (user)name; just report `status.getAnyKey()`.

Reply:
Why `status.getAnyKey()` is not included any more?